cmd/ncdu/ncdu.go

```diff
@@ -42,10 +42,19 @@ builds an in memory representation.  rclone ncdu can be used during
 this scanning phase and you will see it building up the directory
 structure as it goes along.
 
-Here are the keys - press '?' to toggle the help on and off
+You can interact with the user interface using key presses,
+press '?' to toggle the help on and off
 
     ` + strings.Join(helpText()[1:], "\n    ") + `
 
+Listed files/folders may be prefixed by a one-character flag,
+some of them combined with a description in brackes at end of line:
+
+    ! an error occurred while reading this directory
+    . an error occurred while reading a subdirectory, so the indicated size may be underestimated
+    e empty directory, i.e. contains no files but may contain empty subdirectories
+	~ the backend did not report size for all files, so the indicated size may be underestimated
```

Comment:
Typo, tab instead of 4 spaces.
```suggestion
    ~ the backend did not report size for all files, so the indicated size may be underestimated
```

Reply:
Well spotted. Fixed it. Also splittes long lines.